README.md

```diff
@@ -457,30 +605,39 @@ function safe_div(a : Number, b : Number): Number {
 
 Here, `a` and `b` are statically checked to be numbers, so we don't need to
 check their types at runtime. Writing `safe_div("foo","bar")` would be a
-compile-time error and never go undetected to production. Problem is, we still
-need to check if `b` is zero. That's because the type-system of TypeScript (and
-most statically typed languages) has an expressivity barrier that doesn't allow
-it to reason about values of computations. Fixing that lack of expressivity is
-the main benefit of merging types and values on the same level. In Formality,
-we could write:
+compile-time error and never go into production undetected. The problem is, we
+still need to check if `b` is zero. That's because the type-systems of TypeScript
+as well most other statically typed languages have an expressivity barrier that
+doesn't allow it to reason about the values of computations. Fixing that lack of
+expressivity is the main benefit of "first-class types" that allow types to be
+treated as values.
```

Comment:
I'd remove the part after "first-class types", looks redundant at this point

Reply:
agreed